scripts/o3de/o3de/android_support.py

```diff
@@ -74,6 +74,11 @@
 ANDROID_SETTINGS_FILE = '.command_settings'
 ANDROID_SETTINGS_SECTION_NAME = 'android'
 
+# CARBONATED -- begin : asset pack definitions
+AAB_ASSET_PACK_NAME = 'app_asset_pack'
```

Comment:
Let's rename this to AAB_ASSET_PACK_DELIVERY_NAME

Justification:  Makes it clearer as to the specific purpose of the name.  "AAB_ASSET_PACK_NAME" is too generic and could refer/imply other usage, but _DELIVERY_ ties it to AAB_ASSET_PACK_DELIVERY_TYPE, and makes the usage specific and clear.

Reply:
Renamed (this and related)